test/gui/shared/steps/steps.py

```diff
@@ -1466,3 +1463,71 @@ def step(context, username, foldername):
 @Then("credentials wizard should be visible")
 def step(context):
     waitForObject(AccountConnectionWizard.CREDENTIAL_PAGE)
+
+
+@When("user selects configure_synchronization_manually option in advanced section")
+def step(context):
+    clickButton(
+        waitForObject(
+            names.syncModeGroupBox_configureSyncManuallyRadioButton_QRadioButton
+        )
+    )
+    clickButton(waitForObject(names.setupWizardWindow_nextButton_QPushButton))
+
+
+@Then("dialog add folder sync connection should be visible")
+def step(context):
+    mouseClick(
+        waitForObject(
+            names.add_Folder_Sync_Connection_FolderWizardSourcePage_OCC_FolderWizardLocalPath
+        ),
+        271,
+        92,
+        Qt.NoModifier,
+        Qt.LeftButton,
+    )
+    clickButton(
+        waitForObject(
+            names.add_Folder_Sync_Connection_qt_passive_wizardbutton1_QPushButton
+        )
+    )
+
+
+@Then("dialog select a remote destination folder should be visible")
+def step(context):
+    mouseClick(
+        waitForObject(names.add_Folder_Sync_Connection_groupBox_QGroupBox),
+        125,
+        25,
```

Comment:
what does this co-ordinate implies? Can we click on the coordinate (0,0) instead?